website/docs/troubleshooting.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "mongodbatlas"
+page_title: "Provider: MongoDB Atlas"
+sidebar_current: "docs-mongodbatlas-troubleshooting"
+description: |-
+  The MongoDB Atlas provider is used to interact with the resources supported by MongoDB Atlas. The provider needs to be configured with the proper credentials before it can be used.
+---
+
+# Troubleshooting
+
+The following are some of the common issues/errors encountered when using Terraform Provider for MongoDB Atlas :
+
+
+## Issue: Order of element objects in a TypeList attribute randomly changes everytime user runs `terraform plan`:
+
+### Cause:
+This problem occurs if the user is trying to dynamically add object to an attribute List, for example, by using `dynamic`. This is a known Terraform behavior as `dynamic` can attempt bring objects into schema in any order. 
+
+This can be resolved by:
+
+1. Defining a static list of objects in your resource as shown in the example below:
+
+```
+resource "mongodbatlas_advanced_cluster" "main" {
+  name         = "advanced-cluster-1"
+  project_id   = "64258fba5c9...e5e94617e"
+  cluster_type = "REPLICASET"
+
+  replication_specs {
+    region_configs {
+      electable_specs {
+        instance_size = "M20"
+        node_count    = 1
+      }
+      provider_name = "AWS"
+      priority      = 7
+      region_name   = "US_EAST_1"
+    }
+
+    region_configs {
+      electable_specs {
+        instance_size = "M20"
+        node_count    = 1
+      }
+      provider_name = "AWS"
+      priority      = 6
+      region_name   = "EU_WEST_1"
+    }
+  }
+}
+```
+
+2. User can try to maintain order when using `dynamic` by using a `type = list()` variable as shown in the example below:
```

Comment:
Suggest reframing this as the second step in a procedure/ using the same language as step 1
```suggestion
2. Using a `type = list()` variable when using `dynamic` as shown in the example below:
```